resources/windows/wix/project.wxs.erb

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="windows-1252"?>
+  <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+    <Product
+      Id="*"
+      UpgradeCode="<%= settings[:upgrade_code] %>"
+      Name="<%= settings[:product_name] %>"
+      Language="1033"
+      Codepage="1252"
+      Version="<%= @version.sub(/\.g[0-9a-z]{7}/, '') %>"
+      Manufacturer="<%= settings[:company_name] %>"
+    />
+    <Package
+      InstallerVersion="300"
+      InstallScope="perMachine"
+      Description="<%= "#{settings[:product_word]}#{@platform.architecture == "x64" ? " (64-bit)" : ""}" %> Installer"
+      Comments="<%= @homepage %>"
+      Compressed="yes"
+      Platform="<%= @platform.architecture %>"
+    />
+    <MajorUpgrade AllowDowngrades="yes" />
+    <Media Id="1" Cabinet="<%= settings[:product_word] %>.cab" EmbedCab="yes" CompressionLevel="high" />
+
+    <Feature Id="<%= settings[:product_word] %>Runtime" Title="<%= settings[:product_word] %> Runtime" Level="1">
+      <!-- All heat runs and WiX Fragment files containing Component elements will reference this ComponentGroup -->
+      <ComponentGroup Id="ProductComponentGroup" />
```

Comment:
IMHO we should stick with this. This has to be passed to the heat run as well as used in every other component.

An alternative is to remove this line entirely, and have some other file being responsible for hooking into `Feature`